dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -1233,7 +1233,12 @@ private FileLogHeader newFileLogHeader(boolean trace, LogProviderConfigImpl conf
         if (LoggingConstants.JSON_FORMAT.equals(messageFormat)) {
             isJSON = true;
             String jsonHeader = constructJSONHeader(messageFormat, config);
-            return new FileLogHeader(jsonHeader, trace, javaLangInstrument, isJSON);
```

Comment:
I think it would be more keeping with the initial design to just change line 1233 to 
```
if (!trace && LoggingConstants.JSON_FORMAT.equals(messageFormat)) {
```
... and I believe that would also avoid need for changes to FileLogHeader.java